### PR TITLE
Update subscription-manager & Insights

### PR DESCRIPTION
**Pull request by ptoscano**
See the individual commits for more details.

### CODE REVIEW

**Comment by yselkowitz** [comment]
Please write the placeholders in the new format.

**Comment by ptoscano** [comment]
@yselkowitz anything else that I need to change here?

File: configs/sst_insights.yaml [hunk 1]
```diff
@@ -8,19 +8,56 @@ data:
...
+      - rpm_name: rhc
+        description: rhc
+        dependencies:
+        - policycoreutils-python-utils
+        - yggdrasil
+        - yggdrasil-worker-package-manager
+
+    - srpm_name: rhc-worker-playbook
+      build_dependencies:
+      - rhc
```
Comment (yselkowitz) [inline configs/sst_insights.yaml:39]: Placeholders cannot require other placeholders.

File: configs/sst_insights.yaml [hunk 2]
```diff
@@ -8,19 +8,56 @@ data:
...
+      rpms:
+      - rpm_name: rhc-worker-playbook
+        description: rhc-worker-playbook
+        dependencies:
+        - ansible-core
+        - insights-client
```
Comment (yselkowitz) [inline configs/sst_insights.yaml:45]: Placeholders cannot require other placeholders.

File: configs/sst_insights.yaml [hunk 3]
```diff
@@ -8,19 +8,56 @@ data:
...
+    - yggdrasil-worker-package-manager
+
+    ppc64le:
+    - yggdrasil-worker-package-manager
+
+    s390x:
+    - yggdrasil-worker-package-manager
+
+    x86_64:
+    - yggdrasil-worker-package-manager
```
Comment (yselkowitz) [inline configs/sst_insights.yaml:60]: `arch_packages` is for when a package is only for some architectures.  (It is also not needed just for excluding i686.)  What are you excluding here?

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: configs/sst_insights.yaml (hunk 3 above)
Comment (ptoscano) [reply configs/sst_insights.yaml:60]: ah ok; from what I remember in the past (during the el9 work), `packages` was for noarch packages and `arch_packages` was for arch-specific ones

in this case, I can also fold the `arch_packages` in `sst_subscription_manager-subscription-manager.yaml` to `packages`

**Comment by ptoscano** [comment]
> Placeholders cannot require other placeholders.

maybe `test_config_files.py` could check that? ;)

**Pull request merged**